### PR TITLE
adapted according to seed

### PR DESCRIPTION
**Pull request by galathinius**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License
```
Comment (vadimb) [inline LICENSE]: Remove

File: app/app.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const app = angular.module('myApp', [
```
Comment (vadimb) [inline app/app.js]: Rename vars e.g myApp.view1 => app.MovieList

File: app/app.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  'myApp.view1',
+
+])
+  . config(['$locationProvider', '$routeProvider', function ($locationProvider, $routeProvider) {
+    $locationProvider.hashPrefix('!');
+
+    $routeProvider.otherwise({ redirectTo: '/' });
+  }])
+  . controller('MainController', function (movieService) {
+    movieService.getData('now_playing')
```
Comment (vadimb) [inline app/app.js]: Refactor into a comp

File: app/configuration.js [hunk 4]
```diff
@@ -4,3 +4,7 @@ const imageURL = 'https://image.tmdb.org/t/p/w500';
 getLink = function (category) {
```
Comment (vadimb) [inline app/configuration.js]: Refactor/remove

File: app/index.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  <div ng-view></div>
+ 
+  <script src="lib/angular/angular.js"></script>
+  <script src="lib/angular-route/angular-route.js"></script>
+  <script src="app.js"></script>
+  <script src="view1/view1.js"></script>
+  <script src="services/movieService.js"></script>
+  <script src="list/list.js"></script>
+  <script src="configuration.js"></script>
+  <!-- <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.8.0/angular.js"> </script> -->
```
Comment (vadimb) [inline app/index.html]: Remove

File: app/list/list.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<h2>list tag</h2>
+    <ul class="flex-container">
+        <li ng-repeat="x in $ctrl.categories" class="flex-item" >
```
Comment (vadimb) [inline app/list/list.html]: rename x

File: app/list/list.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+angular
+  .module('myApp')
+  .component('list', {
+    templateUrl: 'list/list.html',
+    bindings: { category: "@" },
```
Comment (vadimb) [inline app/list/list.js]: Review componentz bindings

File: app/list/list.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+    controller: function ListController(movieService) {
+      this.$onInit = function () {
```
Comment (vadimb) [inline app/list/list.js]: Review comp lifecycle

File: app/services/movieService.js [hunk 9]
```diff
@@ -0,0 +1,9 @@
+app.factory('movieService', ($http) => ({
+  getData(category) {
+    return $http.get(getLink(category))
+      .then((response) => response.data.results)
```
Comment (vadimb) [inline app/services/movieService.js]: Refactort into one then

File: app/view1/view1.js [hunk 10]
```diff
@@ -0,0 +1,12 @@
+angular.module('myApp.view1', ['ngRoute'])
```
Comment (vadimb) [inline app/view1/view1.js]: Reanme vars

File: app/view1/view1.spec.js [hunk 11]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+describe('myApp.view1 module', function() {
```
Comment (vadimb) [inline app/view1/view1.spec.js]: Add a few tests

**Review by vadimb — APPROVED** [review]

File: app/index.html [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body  ng-app="app">
+  <div ng-view></div>
+ 
+  <script src="lib/angular/angular.js"></script>
+  <script src="lib/angular-route/angular-route.js"></script>
+  <script src="app.js"></script>
+  <script src="MovieLists/MovieLists.js"></script>
+  <script src="services/movieService.js"></script>
+  <!-- <script src="list/list.js"></script> -->
```
Comment (galathinius) [inline app/index.html:17]: to uncomment

File: app/MovieLists/MovieLists.js [hunk 13]
```diff
@@ -0,0 +1,8 @@
+angular.module('app.MovieLists', ['ngRoute'])
+  .config(['$routeProvider', function ($routeProvider) {
+    $routeProvider.when('/', {
+      templateUrl: 'MovieLists/MovieLists.html',
```
Comment (vadimb) [inline app/MovieLists/MovieLists.js:4]: Review naming conventions

File: app/MovieLists/MovieLists.js [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
+      controller: 'MovieListsCtrl',
+    });
+  }])
+  .controller('MovieListsCtrl', [function () { }]);
```
Comment (vadimb) [inline app/MovieLists/MovieLists.js:8]: Why do we need it here?
